addons/project/tests/test_project_task_type.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+from odoo.exceptions import UserError
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+
+
+class TestProjectTaskType(TestProjectCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestProjectTaskType, cls).setUpClass()
+
+        cls.stage_created = cls.env['project.task.type'].create({
+            'name': 'Stage Already Created',
+        })
+
+    def test_create_stage(self):
+        '''
+        Verify that it is not possible to add to a newly created stage a `user_id` and a `project_ids`
+        '''
+
```

Comment:
IMO, the new line here is not really necessary.
```suggestion
        '''
```